Fixing small bugs.

xubonnie opened the pull request:
I made 3 changes:

1) Add picture of graph visualization to README file. When people come across this repository, it's a nice easy way for them to determine if this is something they're interested in. It saves a lot of their time since it avoids having to make a bunch of clicks and navigate too much.

2) Fixed maps referring issue, from line `multistreamDictionary.get("TRANSPORT").get("PACKET_SENT")`, for example. If you'd get this error: `Uncaught TypeError: Cannot read property 'get' of undefined` if TRANSPORT wasn't the category, or PACKET_SENT wasn't the event. 

3) Fixed issue where `packetsSentSizeLUT` would error because it's an empty array (so for example, if there are no packet sent events in the qlog). Line of code is: `scatter.set("ymin", (packetsSentSizeLUT[0].from < 5000) ? 0 : packetsSentSizeLUT[0].from );`.

xubonnie committed:
Made small fixes to tool

xubonnie committed:
Made small fixes to tool

xubonnie committed:
Modified README to include photo

xubonnie committed:
Modified README to include photo

xubonnie committed:
Modified README to include graph visualization

xubonnie committed:
Fixed photo referencing

xubonnie committed:
Fixed photo referencing

xubonnie committed:
Forgot right bracket

xubonnie committed:
Forgot right bracket

xubonnie committed:
Fix formatting on README

xubonnie committed:
Fix formatting on README

xubonnie committed:
Fix formatting on README

xubonnie committed:
Fix formatting on README

xubonnie committed:
Merge branch 'fix-small-stuff' of github.com:xubonnie/quictrace1qlog into fix-small-stuff

rmarx commented on standalone/graph.js:
```diff
@@ -102,9 +101,12 @@ function drawGraph(qlog, settings){
-			let packetsSent = multistreamDictionary.get("TRANSPORT").get("PACKET_SENT") || []; // || [] defaults to an empty array if there are no events of that type present in the log
-			let packetSentList = [];
+			let packetsSent = [];
+		   let packetSentList = [];
```
Could you add another tab before this to align it with the rest?

rmarx commented on standalone/graph.js:
```diff
@@ -128,70 +130,77 @@ function drawGraph(qlog, settings){
-								continue;
-							}
-                            
+			let packetsReceived =[];
```
similar remarks here: did you use the same indentation method as the rest of the file? Looks garbled, at least on github.

xubonnie replied:
I'm not sure why it looks so strange on Github, it looks fine to me on my IDE. 
![image](https://user-images.githubusercontent.com/17273720/61474477-99350a00-a93d-11e9-9472-3622c472a40d.png)

I followed the same indentation method.

rmarx commented:
Thank you for the changes! The code in itself looks good, but at least on Github the indentations look very garbled. Did you switch to tabs vs spaces or 4 vs 2 spaces or something? 

If you could update that, that would be great :)

rmarx committed:
Add query parameter to load from URL

- Adds query parameter ?file= to load from URL

- Has a saner xmax default

- packets now have minimum size of 3px instead of 1px. Still not stellar, should really figure out how to best solve this (difficult because you will have overlap between data and closely-arriving acks)

rmarx committed:
Make packet width scaling better when zooming in

xubonnie committed:
Fix formatting

xubonnie committed:
Fix merge conflict

xubonnie committed:
Update graph.js

xubonnie commented:
Rebased. Not sure why on Github the indentation is so off :(
